backend/src/lib/services/spotify/index.spec.js

```diff
@@ -0,0 +1,22 @@
+import SpotifyService from './index'
+
+jest.mock('../../../config/winston')
+
+const mopidy = {
+  tracklist: {
+    nextTrack: jest.fn()
+      .mockImplementationOnce(() => Promise.resolve('data'))
+      .mockImplementationOnce(() => Promise.resolve())
+  }
+}
+
+describe('SpotifyService', () => {
+  const mockFn = jest.fn()
+
+  describe('canRecommend', () => {
+    it('should handle both cases', () => {
+      SpotifyService.canRecommend(mopidy, mockFn)
+      SpotifyService.canRecommend(mopidy, mockFn)
```

Comment:
Does this need expectations?

Reply:
Yes it does, doh! I need to have a think about testing in general as it seems that Enzyme is proving harder to work with as it's not keeping up with React. For instance you can't test hooks correctly currently.